tests/libsecret/secret.go

```diff
@@ -35,6 +40,27 @@ func New(name string, data map[string][]byte) *kubev1.Secret {
 				secretLabel: name,
 			},
 		},
-		Data: data,
+		Data:       data.Data(),
+		StringData: data.StringData(),
```

Comment:
Can you please add a reference to the doc saying `Data` or `StringData` can be used to pass the data.